commodore/package/compile.py

```diff
@@ -63,7 +67,7 @@ def compile_package(
         cfg,
         targets,
         search_paths=[cfg.vendor_dir],
-        fetch_dependencies=True,
+        fetch_dependencies=cfg.fetch_dependencies,
```

Comment:
This can be dropped. The value can be access from the config object passed to the called context.

Reply:
Will be done in #508